### PR TITLE
Feature/predevelop3

### PR DESCRIPTION
**Pull request by bandurski-maciej**

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: web/src/main/java/com/infoshare/academy/highfive/cdi/FileUploadProcessor.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+@RequestScoped
+public class FileUploadProcessor {
+
+    private String TEMP = "/tmp/hfv/";
+
+//    public File uploadJsonFile(Part filePart) throws IOException, JsonFileNotFound {
```
Comment (rafalmisiak) [inline web/src/main/java/com/infoshare/academy/highfive/cdi/FileUploadProcessor.java]: skasujmy to

File: web/src/main/java/com/infoshare/academy/highfive/dto/view/VacationView.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+  private Long employeeId;
+
+  private String firstName;
+
+  private String surname;
+
+  private String position;
+
+  private LocalDate vacationFrom;
```
Comment (rafalmisiak) [inline web/src/main/java/com/infoshare/academy/highfive/dto/view/VacationView.java]: Dajmy @JsonProperty aby zniknęły camel case

File: web/src/main/java/com/infoshare/academy/highfive/mapper/entity/TeamMapper.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import com.infoshare.academy.highfive.dto.request.TeamRequest;
+
+import javax.enterprise.context.RequestScoped;
+
+@RequestScoped
+public class TeamMapper {
+
+    public Team mapRequestToEntity(TeamRequest request){
+
+   Team team = new Team();
```
Comment (rafalmisiak) [inline web/src/main/java/com/infoshare/academy/highfive/mapper/entity/TeamMapper.java]: formatowanie

File: web/src/main/java/com/infoshare/academy/highfive/scheduler/VacationEntitlementBean.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import java.util.logging.Logger;
+
+@Singleton
+@Startup
+public class VacationEntitlementBean {
+
+  private static final Logger logger =
+    Logger.getLogger(VacationEntitlementBean.class.getName());
+
+//  @EJB
```
Comment (rafalmisiak) [inline web/src/main/java/com/infoshare/academy/highfive/scheduler/VacationEntitlementBean.java]: usuńmy to

File: web/src/main/java/com/infoshare/academy/highfive/service/configuration/UploadJsonService.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    private Logger logger = LoggerFactory.getLogger(getClass().getName());
+
+    @Inject
+    FileUploadProcessor fileUploadProcessor;
+
+    @Inject
+    HolidayService holidayService;
+
+//    public Integer uploadJsonHoliday(Part fileName) throws IOException, JsonFileNotFound {
+//
```
Comment (rafalmisiak) [inline web/src/main/java/com/infoshare/academy/highfive/service/configuration/UploadJsonService.java]: komentarz

File: web/src/main/java/com/infoshare/academy/highfive/web/servlet/Blank.java [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+        dataModel.put("method", req.getMethod());
+        dataModel.put("contentTemplate", "blank.ftlh");
+        dataModel.put("title", "Vacation Manager");
+
+        try {
+            template.process(dataModel, writer);
+        } catch (
+                TemplateException e) {
+            e.getMessage();
```
Comment (rafalmisiak) [inline web/src/main/java/com/infoshare/academy/highfive/web/servlet/Blank.java]: logger

Comment (rafalmisiak) [reply web/src/main/java/com/infoshare/academy/highfive/web/servlet/Blank.java]: logger.error(e.getMessage(),e);

File: web/src/main/java/com/infoshare/academy/highfive/web/servlet/LoginServlet.java [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    String user = (String) req.getAttribute("authorization");
+
+    model.put("role", role);
+    model.put("authorization", user);
+
+    try {
+      template.process(model, writer);
+    } catch (TemplateException e) {
+      logger.warn(e.getMessage());
+      logger.debug("aaaa");
```
Comment (rafalmisiak) [inline web/src/main/java/com/infoshare/academy/highfive/web/servlet/LoginServlet.java]: :)

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**